datasets/nl/nl.full.ad

```diff
@@ -127,8 +127,8 @@
 - offshore_net_costs_present = 26607183.6
 - annual_infrastructure_cost_gas = 1452
 - number_of_cars = 7979083
-- electric_vehicle_profile_1_share = 1.0
-- electric_vehicle_profile_2_share = 0.0
-- electric_vehicle_profile_3_share = 0.0
-- electric_vehicle_profile_4_share = 0.0
-- electric_vehicle_profile_5_share = 0.0
+- electric_vehicle_profile_1_share = 1
```

Comment:
Small thing: why have the 0.0 been replaced by 0? Using integers might give problems when dividing...

Reply:
Good point. This happens automatically when exporting a national dataset. I'll try and fix this in the analysis.